hack/setup-kind-in-ci.sh

```diff
@@ -290,12 +290,44 @@ setup_kind_tempo() {
   infomsg "Installing tempo"
   ${SCRIPT_DIR}/istio/tempo/install-tempo-env.sh -c kubectl -ot true
 
-  infomsg "Installing istio"
-  if [[ "${ISTIO_VERSION}" == *-dev ]]; then
-    local hub_arg="--image-hub default"
+  if [ "${SAIL}" == "true" ]; then
+
+  kubectl create ns istio-system
+  kubectl apply -f https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml
+  kubectl wait pods --all -n opentelemetry-operator-system --for=condition=Ready --timeout=5m
+
+  kubectl apply -f ${SCRIPT_DIR}/istio/tempo/otel-collector.yaml
+
+  infomsg "Installing Istio with Sail"
+
+  local patch_file
+     patch_file=$(mktemp)
+     cat <<EOF > "$patch_file"
+spec:
+  values:
+    global:
+      meshID: mesh-default
+      network: network-default
+      multiCluster:
+        clusterName: cluster-default
+    meshConfig:
+      extensionProviders:
+      - name: otel-tracing
+        opentelemetry:
+          port: 4317
+          service: otel-collector.istio-system.svc.cluster.local
```

Comment:
Riffing on the tempo addon idea from above... maybe you could move this logic into `install-istio-via-sail` as well. All you need to change for tempo is the service right? Something like:
```
if "tempo" in addons:
  ISTIO_YAML = $(yq spec.values.global.meshConfig.extensionProviders[0].service=...)
```

Reply:
Done!